packages/cubejs-backend-native/src/transport.rs

```diff
@@ -377,38 +377,43 @@ impl TransportService for NodeBridgeTransport {
                     continue;
                 }
             }
+
             let response: serde_json::Value = result?;
+
             #[cfg(debug_assertions)]
             trace!("[transport] Request <- {:?}", response);
             #[cfg(not(debug_assertions))]
             trace!("[transport] Request <- <hidden>");
 
-            let load_err = match serde_json::from_value::<TransportLoadResponse>(response.clone()) {
```

Comment:
serde_json::from_value requires passing the own of the value; it's why we used `clone` here. The biggest problem is that we used it for successful responses, which is a large structure.

PR fixes that issue.